<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html lang="ja-JP">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=Shift_JIS">
<meta http-equiv="Content-Style-Type" content="text/css">
<meta name="keywords" content="Simple Directmedia Layer SDL2.0">
<link rel="top" href="index.html" title="ホーム">
<link rel="parent" href="CategoryError.html" title="1つ上">
<title>SDL_GetError</title>
</head>
<body>
<a href="index.html">目次</a> - <a href="ApiByCategory.html">API(機能別)</a> - <a href="CategoryError.html">エラー処理</a> - SDL_GetError
<hr>
<h1>SDL_GetError</h1>
最後に発生したエラーのメッセージを得る
<h2>構文</h2>
<code>const char* SDL_GetError(void)</code>
<h2>戻り値</h2>
<p>
発生したエラーの情報のメッセージを戻す.
または, <a href="SDL_ClearError.html">SDL_ClearError()</a>を呼んでからエラーがなければ, 空の文字列を戻す.
<a href="SDL_ClearError.html">SDL_ClearError()</a>を呼んだ場合を除けば, メッセージはSDL関数がエラーを報告した場合のみ変わる.
<a href="SDL_GetError.html">SDL_GetError()</a>を呼ぶかどうかは, SDL関数の戻り値を確認して決める必要がある.
</p>
<p>
文字列は静的に確保されているので, アプリケーションは解放してはならない.
</p>
<h2>詳細</h2>
<a href="SDL_GetError.html">SDL_GetError()</a>を呼ぶ前に複数のエラーが発生する可能性がある.
この場合, 最後のエラーメッセージのみ戻す.
<h2>関連項目(関数)</h2>
<a href="SDL_ClearError.html">SDL_ClearError</a><br>
<a href="SDL_SetError.html">SDL_SetError</a><br>
<h2>SDL Wikiへのリンク</h2>
<a href="https://wiki.libsdl.org/SDL_GetError">SDL_GetError - SDL Wiki</a>
<hr>
</body>
</html>
